.github/workflows/test-kms.yml

```diff
@@ -0,0 +1,61 @@
+name: Run KMS tests
+
+on:
+  push:
+  workflow_dispatch:
+
+permissions: {}
+
+jobs:
+  test-kms:
+    runs-on: ubuntu-latest
+
+    permissions:
+      id-token: 'write' # for OIDC auth for GCP authentication
+      issues: 'write' # for filing an issue on failure
+
+    steps:
+      - name: Checkout securesystemslib
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          cache: 'pip'
+          cache-dependency-path: 'requirements*.txt'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade tox
+
+      - name: Authenticate to Google Cloud
+        uses: google-github-actions/auth@c4799db9111fba4461e9f9da8732e5057b394f72
+        with:
+          token_format: access_token
+          workload_identity_provider: projects/843741030650/locations/global/workloadIdentityPools/securesystemslib-tests/providers/securesystemslib-tests
```

Comment:
Do you need to adopt the service name change (f48786f182e5ddf729d2fba7f7fdff3d21225ab3) in the `workload_identity_provider` path? (I did not check if or how they are related)

Reply:
I think no: service_account is the actual GCP service account that has the permissions to sign. workload_identity_provider is what allows our Github OIDC identity to act as the service account. So two different things that are only connected by "service account impersonation", not by name.